app/src/main/java/com/example/sharingang/EditItemFragment.kt

```diff
@@ -77,7 +77,7 @@ class EditItemFragment : Fragment() {
                     categoryString = resources.getStringArray(R.array.categories)[binding.categorySpinner.selectedItemPosition],
                     latitude = binding.latitude?.toDoubleOrNull() ?: 0.0,
                     longitude = binding.longitude?.toDoubleOrNull() ?: 0.0,
-                    imageUri = imageUri?.toString()
+                    imageUri = if(imageUri != null) imageUri.toString() else existingItem.imageUri
```

Comment:
```suggestion
                    imageUri = imageUri?.toString() ?: existingItem.imageUri
```

Pretty sure that this works, and is more idiomatic kotlin

Reply:
Yeah you're right, I tried to do something like that but failed at first, thanks!